tests/test_downloads.py

```diff
@@ -189,3 +189,11 @@ def test_download_with_redirect_original_url_used_for_filename(self, httpbin):
             assert os.listdir('.') == [expected_filename]
         finally:
             os.chdir(orig_cwd)
+
+    def test_download_quietflag(self, httpbin_both, httpbin):
+        robots_txt = '/robots.txt'
+        body = urlopen(httpbin + robots_txt).read().decode()
+        env = MockEnvironment(stdin_isatty=True, stdout_isatty=False)
+        r = http('--quiet', '--download', httpbin_both.url + robots_txt, env=env)
+        assert r.stderr == ''
+        assert body == r
```

Comment:
you should also test that both `env.std(out|err)` get replaced with `env.devnull`